### ISSUE TITLE
2 things that can make C# interactive more powerful.

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/roslyn#26450]
Currently C# interactive work perfect but it's need a little enhancement so it can be more great.

Let's make C# interactive great again.

1. If I have RestCleint in my project (for example)

I must be able to write

#r RestClient or dll name to register the package. This command will find dll in bin folder and register it. currently typing full path a little bit hard to follow.

2. User should be able to use all the namespace that is aviliable on the project for example

using System.Text.RegularExpressions; 

this will work but not if I enter restclient. 

If any doubt, let me know I can help on giving more detail on this improvement.

_This issue has been moved from https://developercommunity.visualstudio.com/content/problem/241423/2-things-that-can-make-c-interactive-more-powerful.html
VSTS ticketId: 607191_
_These are the original issue comments:_
(no comments)
_These are the original issue solutions:_
(no solutions)

**Comment by anirugu** [comment]
If anyone needs some help on reproducing things, let me know. I can give you more details.

**Comment by CyrusNajmabadi** [comment]
Not planning on investing here.
